components/EditButton/EditButton.styled.js

```diff
@@ -0,0 +1,13 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export const StyledEditButton = styled(Link)`
+  background-color: #90ee90;
+  margin: 0em;
+  padding: 0 6px;
+  border-style: outset;
+  border-color: buttonborder;
+  font-size: 13px;
+  text-decoration: none;
+  color: #ffffff;
```

Comment:
**Here a some suggestions 🙃** 

-  **Mixing units** can be a source of inconsistency in your designs. The code uses both` px` and `em` for spacing. It's generally better to stick to one unit for consistency unless there's a specific reason for mixing them.
- `border-color: buttonborder`:  seems like a typo or an undeclared variable

Reply:
Hey Jess, we changed the border-color situation, thank you for pointing it out! In general, this styling will likely be completely overwritten with US6 and the new styling, so we wont change the units right now but definitely keep them in mind for when we rewrite the styling! :D